spaces/S000045/properties/P000016.md

```diff
@@ -3,10 +3,10 @@ space: S000045
 property: P000016
 value: true
 refs:
-- doi: 10.1007/978-1-4612-6290-9_6
+- zb: "0386.54001"
   name: Counterexamples in Topology
 ---
 
-Given any open cover of $X$, any two sets containing -1 and 1 must cover $X$.
+Given any open cover of $X$, the two sets containing -1 and 1 must cover $X$.
```

Comment:
```suggestion
Given any open cover of $X$, any two sets containing -1 and 1 must cover $X$.
```
"Any" was fine here.  We cannot use the definite article "the", because there could be many sets in the cover containing $1$ and $-1$.  And also, no two such sets have been singled out in a previous sentence.  So in the current context there is no definite set of two such sets.